content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -661,6 +666,11 @@ the managedFields, this will result in the managedFields being reset first and
 the other changes being processed afterwards. As a result the applier takes
 ownership of any fields updated in the same request.
 
+### Known Issues
+
+- There is a known issue with using Server Side Apply on sub-resource endpoints
+  where field ownership is not being updated [#88981](https://github.com/kubernetes/kubernetes/issues/88981). 
+
```

Comment:
```suggestion
{{< caution >}}
Server Side Apply does not correctly track ownership on sub-resources. If you are using
Server Side Apply with a subresource, …

{{< /caution >}}
```

what should the `…` say?

Reply:
(written to [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future))